src/routes/chat/Chat.js

```diff
@@ -46,7 +45,7 @@ function Chat() {
     event => {
       event.preventDefault();
       const newMessage = {
-        user: testUserId,
+        UserId: testUserId,
```

Comment:
Why do you use PascalCase in a name of a field here?

Reply:
There was something trouble on server side and I must chenge user for UserId, that my model of user and scop works. Accosiation nedeed exactly feald UserId.